Tasks/e1_depth_first_search.py

```diff
@@ -10,5 +11,13 @@ def dfs(g: nx.Graph, start_node: Hashable) -> List[Hashable]:
 	:param start_node: starting node of search
 	:return: list of nodes in the visited order
 	"""
-	print(g, start_node)
-	return list(g.nodes)
+	visited = []
+	st.push(start_node)
+	while st.leng() > 0:
+		start_node = st.pop()
+		visited.append(start_node)
+		neighbors = list(g.neighbors(start_node))
```

Comment:
```suggestion
		neighbors = g.neighbors(start_node)
```
See comments to bfs
non blocking